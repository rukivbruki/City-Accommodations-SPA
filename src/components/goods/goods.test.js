import React from "react";
import renderer from "react-test-renderer";
import Goods from "./goods";

it(`Goods component render correct`, () => {
  const goods = [
    `Wi-Fi`,
    `Washing machine`,
    `Towels`,
    `Heating`,
    `Kitchen`
  ];

  const component = renderer
    .create(<Goods goods={goods}/>)
    .toJSON();

  expect(component).toMatchSnapshot();
});
