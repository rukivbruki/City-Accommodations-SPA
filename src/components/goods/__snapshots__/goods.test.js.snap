// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Goods component render correct 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
  >
    Wi-Fi
  </li>
  <li
    className="property__inside-item"
  >
    Washing machine
  </li>
  <li
    className="property__inside-item"
  >
    Towels
  </li>
  <li
    className="property__inside-item"
  >
    Heating
  </li>
  <li
    className="property__inside-item"
  >
    Kitchen
  </li>
</ul>
`;
